source/reference/minio-server/settings/core.rst

```diff
@@ -295,3 +295,39 @@ Defaults to ``"text/*, application/json, application/xml, binary/octet-stream"``
    Some	types of files cannot be significantly reduced in size.
    MinIO will *not* compress these, even if specified in an :mc-conf:`~compression.mime_types` argument.
    See :ref:`Excluded types <minio-data-compression-excluded-types>` for details.
+
+Erasure Stripe Size
+~~~~~~~~~~~~~~~~~~~
+
+.. tab-set:: 
+
+   .. tab-item:: Environment Variable
+      :sync: envvar
+
+      .. envvar:: MINIO_ERASURE_SET_DRIVE_COUNT
+
+   .. tab-item:: Configuration Variable
+      :sync: config
+
+      .. include:: /includes/common-mc-admin-config.rst
+         :start-after: start-minio-settings-no-config-option
+         :end-before: end-minio-settings-no-config-option
+
+*Optional*
+
+The :ref:`erasure set size <minio-ec-basics>` to apply for all drives in a given :term:`server pool`.
+
+You **must** set this value before you initialize the cluster.
```

Comment:
Saying `must` here just below the `optional` is odd.

Perhaps
```suggestion
If you change to this value, you **must** do so *before* you initialize the cluster.
```

Reply:
```suggestion
If you set this value, you **must** do so *before* you initialize the cluster
```

Slight tweak since it's not a 'change' per-se, and we don't support 'changing' it later. But yes.